### ISSUE TITLE
Turn off insights logging

### ISSUE THREAD

**Issue by mariuscristescuascend** [algolia/instantsearch-ios#310]
**Is your feature request related to a problem? Please describe 🙏**

I recently integrated the InstantSearch SDK v7.26.1 and AlgoliaSearchClient SDK v8.19.0  in our iOS app in order to trigger several InstantSearch Insights events.

After I finished my integration debugging I was left with a verbose logging in my console that I cannot turn off. Every 30 seconds I get a log like this:
> 2024-03-06T14:37:01+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing
> 2024-03-06T14:37:31+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing
> 2024-03-06T14:38:01+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing
> 2024-03-06T14:38:31+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing
> 2024-03-06T14:39:01+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing
> 2024-03-06T14:39:31+0200 info Insights (B5UR3QDMCF) : [InstantSearchInsights] no pending event packages, skip flushing

I instantiate Insights with:
>         Insights.register(appId: ApplicationID(rawValue: algoliaAppId),
>                          apiKey: APIKey(rawValue: algoliaApiKey),
>                          userToken: UserToken(rawValue: String(user.id)))

I tried this, but it had no effect:
>        Insights.shared(appId: ApplicationID(rawValue: algoliaAppId))?.isLoggingEnabled = false

This logging past the integration phase is unnecessary and pollutes the console, making it hard to see important logging.

*Describe the solution you'd like 🤔**

Provide a way to turn of the logging, or better yet, make logging default to silent and only activate it when needed.

**Comment by squarefrog** [comment]
Yeah this is super annoying. You can disable this with `Logs.logSeverityLevel = .critical`, but it should really be fixed!

**Comment by afrencalg** [comment]
Addressed in v7.26.2 with the isAutoSendingHitsViewEvents flag (#314).
